src/client-ocurl/opentelemetry_client_ocurl.mli

```diff
@@ -3,12 +3,6 @@
    https://opentelemetry.io/docs/reference/specification/protocol/exporter/
    *)
 
-val get_url : unit -> string
-
-val set_url : string -> unit
-(** Url of the endpoint. Default is "http://localhost:4318",
-    or "OTEL_EXPORTER_OTLP_ENDPOINT" if set. *)
```

Comment:
Do we want to be able to change `urls` after setup? If so, I can re-introduce `set_url_{traces,metrics,logs}`.

For getters, I'm wondering if that's needed, one can always do:

```ocaml
let config = ref (Config.make ())

(* and then somewhere else *)
let url_traces = !config.url_traces
```

Or am I missing something ?